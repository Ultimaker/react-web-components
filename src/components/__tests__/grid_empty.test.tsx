// Copyright (c) 2018 Ultimaker B.V.
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import GridEmpty from '../grid_empty';

describe('The GridEmpty component', () => {
    let props;
    let wrapper;

    beforeEach(() => {
        props = {
            label: 'Empty grid',
        };
        wrapper = shallow(<GridEmpty {...props} />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
