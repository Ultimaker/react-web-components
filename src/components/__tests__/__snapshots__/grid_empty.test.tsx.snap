// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The GridEmpty component should render 1`] = `
<GridItem
  breakpoint="xs"
  layoutWidth="1/1"
>
  <div
    className="grid-component__empty"
  >
    Empty grid
  </div>
</GridItem>
`;
